src/main/java/seedu/address/logic/commands/update/UpdateCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.commands.update;
+
+import java.util.Set;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.model.name.Name;
+import seedu.address.model.property.Address;
+import seedu.address.model.property.Deadline;
+import seedu.address.model.property.PostalCode;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.Type;
+import seedu.address.model.property.client.Client;
+import seedu.address.model.property.status.Status;
+import seedu.address.model.remark.Remark;
+import seedu.address.model.tag.Tag;
+
+public abstract class UpdateCommand extends Command {
+
+    public static final String COMMAND_WORD = "update";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the status of a property in the app. \n"
+            + "Parameters: INDEX "
+            + "new AMOUNT\n"
+            + "or INDEX proceed|cancel";
```

Comment:
Maybe you will like to include an example usage as well

Reply:
oh yeah, nice catch!